### PR TITLE
More Kataphract Stuff

### PR DESCRIPTION
**Pull request by Geevies**
* Gave the Kataphracts a new backpack, the Hegemony Satchel, which has slightly more than normal storage space.
* Tweaked the Kataphract oxygen tank and magboots to fit the aesthetic some more.
* Tweaked the colours of the underclothes the Kataphracts spawn with to be more easy on the eye.
* Gave the Kataphract Klax a set of magboots.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 1]
```diff
@@ -36,6 +36,10 @@
 	desc = "A tank of oxygen, this one is red."
 	icon_state = "oxygen_fr"
 
+/obj/item/tank/oxygen/brown
+	desc = "A tank of oxygen, this one is brown."
+	icon_state = "oxygen_br"
+
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/tanks/tank_types.dm]: Unnecessary whitespace.

Comment (Geevies) [inline code/game/objects/items/weapons/tanks/tank_types.dm]:
```suggestion
```
